bin/git-overwritten

```diff
@@ -22,6 +22,9 @@ unset head
 unset base
 unset diff_type
 
+# Figure out what argument to use for date as GNU & BSD `date`s are different
+[[ $(date -r123 '+%s' 2>/dev/null) -eq 123 ]] && DATEARG='-r' || DATEARG='-d@'
```

Comment:
Sorry, I just realized that `-eq` (number comparison) was a bad suggestion because if the command fails (like we expect with GNU date), the first argument will be empty string and `-eq` will yield an error because the argument is not a number. It's best to use string comparison `=`.

Also, please wrap both arguments in double quotes.